src/Renderer.cpp

```diff
@@ -0,0 +1,43 @@
+
+#include "Renderer.h"
+
+#include <iostream>
+
+namespace ccm
+{
+	Renderer::Renderer(Application& app)
+	{
+		/* Create renderer */
+		m_renderer = SDL_CreateRenderer(app.getWindow(), -1, SDL_RENDERER_ACCELERATED);
+		if (m_renderer == nullptr)
+		{
+			std::cerr << "SDL_CreateRenderer Error: " << SDL_GetError() << std::endl;
+			app.m_quit = true;
```

Comment:
Should we consider making `Application` a singleton, so for example we can quit from anywhere without having to pass it around everywhere we go?

Reply:
I'm more inclined to have the calling system emit a Quit Signal that would be picked up during the polling on the next frame? Since using a singleton is a relatively drastic change in the design, I would also be inclined to have that as another issue/PR request aside from rendering. It would at least give us some times to brainstorm ideas.